src/servicebus/azext_servicebus/__init__.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+
+import azure.cli.command_modules.servicebus._help  # pylint: disable=unused-import
```

Comment:
This should be `azext_servicebus._help`
`azure.cli.command_modules.servicebus._help` wouldn't exist.